packages/website/app/routes/home.tsx

```diff
@@ -1,36 +1,101 @@
-import type { LinksFunction,MetaFunction } from '@remix-run/server-runtime' 
-
-import { Link, Outlet } from '@remix-run/react'
-
-import stylesUrl from '~/styles/articleBox.css'
-
-
-export const meta: MetaFunction = () => ({
-	title: 'Home',
-})
-
-
-export const handle = {
-	breadcrumb: () => <Link to="/home">Home</Link>,
-}
-export const links: LinksFunction = () => {
-	return [{ rel: 'stylesheet', href: stylesUrl }]
-}
-export default function Home() {
-	return (
-		<div className="home-container w-screen  min-h-screen">
-			<div className="top h-3xl w-full mb-20">
-				<img className="h-full w-full object-cover" src="https://hsiaofongw.notion.site/images/page-cover/rijksmuseum_jansz_1641.jpg" alt="" />
-			</div>
-			<div className="mx-auto min-h-screen" w-w="1024px">
-				<h1 w-text="60px" >探索IOClub首页</h1>
-				<p>MemberList</p>
-				<main>
-					<Outlet />
-				</main>
-			</div>
-			<div className="bottom">
-			</div>
-		</div>
-	)
-}
\ No newline at end of file
+import type {
+	LoaderFunction,
+	MetaFunction,
+} from '@remix-run/server-runtime'
+import type { IArticleBox } from '~/types/articleBox'
+
+import { Link , useFetcher } from '@remix-run/react'
+import { json } from '@remix-run/server-runtime'
+import {useEffect, useState } from 'react'
+
+import { ArticleBox } from '~/components/articleBox/articleBox'
+import { db } from '~/utils/db.server'
+
+export const meta: MetaFunction = () => ({
+	title: 'Home',
+})
+
+export const handle = {
+	breadcrumb: () => <Link to="/home">Home</Link>,
+}
+export const loader: LoaderFunction = async ({ request }) => {
+	const index =
+    parseInt(new URL(request.url).searchParams.get('index') as string) || 0

```

Comment:
Why do we have `index`? Do you want to support paging on an infinite scroll page? If so, I recommended https://remix.run/docs/en/v1/guides/data-loading#route-params. `route params` are easier than `query parameters`.

Reply:
We need infinite scrolling !!! :)

Reply:
if I use params , infinite scrolling will broken :<

Reply:
or we can use /home?index=${index}/pageId

Reply:
Abstract another `resource route` under `/app/routes/api/articles`.

Reply:
Your api design sucks, plz refer graphql pagination model for some general design.